jsk_rviz_plugins/src/pie_chart_display.cpp

```diff
@@ -359,6 +363,11 @@ namespace jsk_rviz_plugins
     
   }
   
+  void PieChartDisplay::updateCaption()
+  {
+    caption_ = caption_property_->getStdString();
```

Comment:
btw, ` caption_property_->getStdString();` is initialized at https://github.com/jsk-ros-pkg/jsk_visualization/pull/636/files#diff-21915262cfae72cf9ac2af03ff4ea8e7R91, so iw you worried about null caption name, then we can set at there.
If you think we do not have to define `caption_`, but can use `name_`, I  do not have confident on modifying `name_` because did not know how is this variable used